CODEOWNERS

```diff
@@ -12,7 +12,7 @@
 # The following filepaths should be sorted so that more specific paths occur
 # after the less specific paths, otherwise the ownership for the specific paths
 # is not properly picked up in Github.
-* @cilium/janitors
+* @cilium/cli
```

Comment:
Agree with the change itself. It looks like we can also remove the comment describing `@cilium/janitors` at the top of the file, given that this group is no longer referenced in `CODEOWNERS` (I cannot comment on the actual line, it seems).

Reply:
👍 will update. let's see if the CI passes again 😄

Reply:
Let's continue your lucky streak with 🟢  CI 😄